### DISCUSSION TITLE
Which home assistant package are you using from Tteck site?

### DISCUSSION THREAD

**Discussion by NorthernMunkey** [tteck/Proxmox#4011] [Help Needed]
little bit of background, i've used hasos, debian with HA in docker before,  planning on moving to proxmox HA, who uses what and why? which one would be the best (complete) option, 
Home Assistant Container LXC
Home Assistant Core LXC
Home Assistant OS VM
PiMox HAOS VM
Podman Home Assistant Container LXC
are the offerings,  what are the pros/cons of each,   I was thinking of running all the extra components as lxc's ( mqtt, zigbee2mqtt etc) but have been advised against, purely from a maintenance perspective at least in HA when updates come out they are simple one clicks. AND I'm informed about them, I'd need to run HACS  ( if that had any bearing on which one i would choose)

thanks for any input people provide

**Comment by ykpdang** [comment]
To add to this, does Home Assistant Container LXC create a separate Docker instance in an LXC just for HA?

**Comment by MickLesk** [comment]
i use the OS VM only (since 1 1/2 year). Its the best maintenance option. Never had problems.

**Comment by Mellowlynx** [comment]
I also use the OS VM.
Works great and maintenance is easy.

**Comment by Vakrehus** [comment]
Home Assistant Core LXC - because I want full control over the system.
At the very beginning with HA, I had the Docker version under OMV with the Supervisor. The Supervisor is a nice thing, but it kept causing problems. I like the lean and high-performance systems, which is why I only considered the Core version.
Other LXCs running MQTT Mosquitto, Zigbee2MQTT, NodeRed, ESPHome, InfluxDB and PostgreSQL in conjunction with Home Assistant.
Docker had its day, I don't want to use it anymore.
